webapp/src/main/java/org/opentestsystem/rdw/admin/model/CsvSubject.java

```diff
@@ -0,0 +1,19 @@
+package org.opentestsystem.rdw.admin.model;
+
+/**
+ * This class represents the Group subjects allowed during CSV Group uploads.
+ */
+public enum CsvSubject {
```

Comment:
Unfortunately, since "All" is a viable subject for the CSV export, I couldn't re-use the org.opentestsystem.rdw.common.model.Subject enum